doc/source/train/examples/torch_data_prefetch_benchmark/benchmark_example.rst

```diff
@@ -0,0 +1,37 @@
+:orphan:
+
+Torch Data Prefetching Benchmark Example
+========================================
+
+We provide a benchmark example to show how the auto pipeline for host to device data transfer speeds up training on GPUs.
```

Comment:
Maybe add a quick code snippet right after here which shows the user how easy it is to enable this feature!

```python
train_loader = train.torch.prepare_data_loader(
    data_loader=train_loader, move_to_device=True, auto_transfer=True
)
```
And also link to the docstring?

Reply:
[This](https://ray--22974.org.readthedocs.build/en/22974/train/examples/torch_data_prefetch_benchmark/benchmark_example.html) is the latest doc build. It looks much better! Thanks a lot for your suggestions!